### PR TITLE
Fonts Callbacks logo margins

### PR DESCRIPTION
**Pull request by HerrBaumhaus**
Adjusted Fontset (H1-H6), added divs and callbacks for machine parameters&status, added logo option, tidied up margins

### CODE REVIEW

File: monfrontend/app.py [hunk 1]
```diff
@@ -130,6 +151,43 @@ def fetch_data(intervals):
...
+    """
+    Generates live machine parameters as children of div 'machine-parameters'
+    """
+    measurements = list(influx.get_measurements())
+    # for now use all available measurements, instead data.keys?
+
+    children = []
+    for msmt in measurements:
+        children.append(
+            html.H6(f"MachineStatus:{1}", className="motor_status"),
```
Comment (JensPfeifle) [inline monfrontend/app.py]: Here you will get as many MachineStatus strings with identical contents ("MachineStatus:1") as there are measurements in the InfluxDB. Is that intended?

File: monfrontend/app.py [hunk 2]
```diff
@@ -130,6 +151,43 @@ def fetch_data(intervals):
...
+)
+def live_machine(data):
+    """
+    Generates live machine parameters as children of div 'machine-parameters'
+    """
+    measurements = list(influx.get_measurements())
+    # for now use all available measurements, instead data.keys?
+
+    children = []
+    for msmt in measurements:
```
Comment (JensPfeifle) [inline monfrontend/app.py]: Here as well, 5 strings per measurement that is in the database

**Review by JensPfeifle — COMMENTED** [review]
TravisCI has some complaints about style, click [here](https://travis-ci.org/github/HDventilator/mon-frontend/jobs/684598701) to see them
```
monfrontend/app.py:55:34: C0303: Trailing whitespace (trailing-whitespace)
monfrontend/app.py:57:0: C0303: Trailing whitespace (trailing-whitespace)
monfrontend/app.py:58:0: C0301: Line too long (109/100) (line-too-long)
monfrontend/app.py:62:71: C0303: Trailing whitespace (trailing-whitespace)
monfrontend/app.py:187:59: C0303: Trailing whitespace (trailing-whitespace)
monfrontend/app.py:188:67: C0303: Trailing whitespace (trailing-whitespace)
monfrontend/app.py:165:8: W0612: Unused variable 'msmt' (unused-variable)
monfrontend/app.py:182:8: W0612: Unused variable 'msmt' (unused-variable)
```

File: monfrontend/app.py (hunk 1 above)
Comment (HerrBaumhaus) [reply monfrontend/app.py]: No that's not intended. Wrong Syntax/placeholder. Needs to be mapped to actual status values transmitted.

File: monfrontend/app.py (hunk 2 above)
Comment (HerrBaumhaus) [reply monfrontend/app.py]: No that's not intended. Wrong Syntax/placeholder. Needs to be mapped to actual status values transmitted.

File: monfrontend/app.py (hunk 1 above)
Comment (JensPfeifle) [reply monfrontend/app.py]: Okay. The selection of which measurements to display (and stuff like the display name and unit) should be set in the configuration (coming soon-ish). How about we just drop the loop for now in these two places (this and line 182) and return just the static placeholders you implemented?

**Pull request merged**